config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,3 @@
+# enable pulpcore
+
+answers['foreman_proxy']['plugin']['pulp']['pulpcore_enabled'] = true if answers['foreman_proxy']
```

Comment:
The key is the class name. You should also be careful that it can be a boolean or a hash. Something like this probably makes sense.

```suggestion
cls = 'foreman_proxy::plugin::pulp'
if answers[cls].is_a?(Hash)
  answers[cls]['pulpcore_enabled'] = true
elsif answers[cls]
  answers[cls] = { 'pulpcore_enabled' => true }
end
```